bin/oay/DEPENDENCIES.md

```diff
@@ -0,0 +1,3 @@
+# Dependencies
+
+Refer to [DEPENDENCIES.rust.csv](./DEPENDENCIES.rust.csv) for full list.
```

Comment:
Perhaps inline in the README.md file.

Reply:
We mentioned `DEPENDENCIES.md` in the top-level `DEPENDENCIES.md`:

> Please visit every package's `DEPENDENCIES.md` for a full list of dependencies.

How about we keep the current hierarchy?